docs/reference/method/MongoDBGridFSBucket-registerGlobalStreamWrapperAlias.txt

```diff
@@ -49,6 +49,7 @@ Supported stream functions:
 - :php:`file() <file>`
 - :php:`fopen() <fopen>` with "r", "rb", "w", and "wb" modes
 - :php:`rename() <rename>` rename all revisions of a file in the same bucket
+- :php:`unlink() <unlink>` (remove all revisions of a file)
```

Comment:
Should "in the same bucket" be mentioned here, as it is for `rename()` above?

I'd consider "in the same bucket" to be assumed, but can understand why you might want to be explicit here.